plugins/source/aws/resources/services/iam/policies.go

```diff
@@ -167,3 +173,22 @@ func resolveIamPolicyVersionDocument(ctx context.Context, meta schema.ClientMeta
 	}
 	return nil
 }
+
+func resolveIamPolicyTags(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	r := resource.Item.(types.Policy)
+	cl := meta.(*client.Client)
+	svc := cl.Services().IAM
+	response, err := svc.ListPolicyTags(ctx, &iam.ListPolicyTagsInput{PolicyArn: r.Arn})
+	if err != nil {
+		if cl.IsNotFoundError(err) {
+			meta.Logger().Debug("ListPolicyTags: role does not exist", "err", err)
+			return nil
+		}
+		return diag.WrapError(err)
+	}
+	tags := map[string]*string{}
+	for _, t := range response.Tags {
+		tags[*t.Key] = t.Value
+	}
+	return diag.WrapError(resource.Set("tags", tags))
```

Comment:
```suggestion
	return diag.WrapError(resource.Set("tags", client.TagsToMap(response.Tags)))
```

You can use the helper function `client.TagsToMap` to handle aggregating the Tags

Reply:
Hey I was  wondering if I could bypass creating my own resolver here and do `client.ResolveTags`? Would that also be a valid solution?

Reply:
So in this case you can't use `client.ResolveTags` because you are doing more than transforming data you have already fetched. You can use that function when the main `Describe` call returns the tags without having to make another API call